paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationKtx.kt

```diff
@@ -28,11 +28,26 @@ internal fun PaymentSheet.Configuration.validate() {
                     " the Customer ID cannot be an empty string."
             )
         }
-        customer?.ephemeralKeySecret?.isBlank() == true -> {
-            throw InvalidParameterException(
-                "When a CustomerConfiguration is passed to PaymentSheet, " +
-                    "the ephemeralKeySecret cannot be an empty string."
-            )
+    }
+
+    customer?.accessType?.let { customerAccessType ->
+        when (customerAccessType) {
+            is PaymentSheet.CustomerAccessType.LegacyCustomerEphemeralKey -> {
+                if (customerAccessType.ephemeralKeySecret.isBlank() || customer.ephemeralKeySecret.isBlank()) {
+                    throw InvalidParameterException(
+                        "When a CustomerConfiguration is passed to PaymentSheet, " +
+                            "the ephemeralKeySecret cannot be an empty string."
+                    )
```

Comment:
`InvalidParameterException` is a `java.security` exception type. Should we switch these to `IllegalArgumentException` instead?

Reply:
Probably.